action.yml

```diff
@@ -14,6 +14,12 @@ inputs:
   access_secret:
     description: "access_secret"
     required: true
+  build1:
```

Comment:
Is there a reason we can't call these **amd64** and **arm64** instead of **build1**/**build2**?

Reply:
it's because in theory we might have some other build targets... so instead of setting 
arm64: "true"
amd64: "true",
hich was my first thought, this allows us to set 
build1: "arm64" 
build2: "amd64" or build2:"armhf"...

I know we don't support anything other than amd64 and arm64 anyway with EdgeX though. 

An example of using this is 

https://github.com/canonical/edgex-sync/blob/62d593f8a7ebdb79c23fdf741624f64bdca098c6/.github/workflows/wf-device-grove.yml#L39-L51